src/core/scalar/bytes.rs

```diff
@@ -14,6 +14,10 @@ impl super::Scalar for Bytes {
         |value| value.as_str().is_some()
     }
 
+    fn validate_generic<Value: JsonLikeOwned>(&self) -> fn(&Value) -> bool {
```

Comment:
if we have validate_generic, then wy do we need `validate`?

Reply:
for current impl with async-graphql..

Reply:
```suggestion
    fn validate<'a, Value: JsonLike<'a>>(&'a self) -> fn(&'a Value) -> bool {
```